ui/widgets/nuxeo-collapsible.js

```diff
@@ -50,7 +50,7 @@ import { html } from '@polymer/polymer/lib/utils/html-tag.js';
    *
    */
 
-  class Collapsible extends Nuxeo.Element {
+  class Collapsible extends mixinBehaviors([], Nuxeo.Element) {
```

Comment:
Were you able to understand what is missing on `Nuxeo.Element`?
This intended to be a workaround and a clue, not the final solution 😅

Reply:
I dig for a while but couldn't find any solid conclusions. Guess I have to explore it further

Reply:
Not sure if we are ok to merge it like this. 🤷‍♂️

Reply:
You mean you can't just extend from `Nuxeo.Element`?

Reply:
What's not working here, exactly? On storybook, it seems to work just fine without the `mixinBehaviors`.